src/components/common/CheckList.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+import { css, useTheme } from '@emotion/react';
+
+import { CheckCircleIcon, ChevronIcon } from './icons';
+
+export interface Props {
+  id: string;
+  isChecked: boolean;
+  onToggle: (checked: boolean) => void;
+  onClick?: VoidFunction;
+}
```

Comment:
```suggestion
export interface CheckListProps {
  id: string;
  isChecked: boolean;
  onToggle: (checked: boolean) => void;
  onClick?: VoidFunction;
}
```

깃헙 기능을 살며시 사용해봅니다...

Reply:
감사합니당 ㅋㅋㅋㅋ